python_modules/dagster/dagster/_core/storage/event_log/sqlite/sqlite_event_log.py

```diff
@@ -368,15 +368,17 @@ def delete_events(self, run_id: str) -> None:
             self.delete_events_for_run(conn, run_id)
 
     def wipe(self) -> None:
-        # should delete all the run-sharded dbs as well as the index db
+        # should delete all the run-sharded db files and drop the contents of the index
         for filename in (
             glob.glob(os.path.join(self._base_dir, "*.db"))
             + glob.glob(os.path.join(self._base_dir, "*.db-wal"))
             + glob.glob(os.path.join(self._base_dir, "*.db-shm"))
         ):
-            os.unlink(filename)
+            if f"{INDEX_SHARD_NAME}.db" not in filename:
```

Comment:
probably a better way to do this part...

Reply:
maybe slightly cleaner with an `endswith` ? i think this is fine to commit but leave a comment here in the code